vpc/README.md

```diff
@@ -0,0 +1,11 @@
+# VPC module
+This creates the VPC for the MDN account
+
+If you need to create a VPC for another region you will need to create a separate folder for the region, this allows us to have separate state files
```

Comment:
I would make the phrase after the comma a separate sentence, so change `for the region, this allows us to have separate state files` to `for the region. This allows us to have separate state files.`